Tasks/DeployVisualStudioTestAgent/task.json

```diff
@@ -1,7 +1,7 @@
 ﻿{
     "id": "52A38A6A-1517-41D7-96CC-73EE0C60D2B6",
     "name": "DeployVisualStudioTestAgent",
-    "friendlyName": "Deploy Visual Studio Test Agent",
+    "friendlyName": "Visual Studio Test Agent Deployment",
     "description": "Deploy and configure Test Agent to run tests on a lab machine group.",
```

Comment:
there was one more change i guess. to move the machine name in environment group. also please run gulp to get loc changes